components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -111,30 +126,40 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
 
     const highlight = bodyText;
 
-    if (isValidPullRequestUrl(pullrequestLink) || isValidIssueUrl(pullrequestLink)) {
+    if (isValidPullRequestUrl(highlightLink) || isValidIssueUrl(highlightLink) || isValidBlogUrl(highlightLink)) {
       // generateApiPrUrl will return an object with repoName, orgName and issueId
       // it can work with both issue and pull request links
-      const { apiPaths } = generateApiPrUrl(pullrequestLink);
-      const { repoName, orgName, issueId } = apiPaths;
-      setLoading(true);
-      // Api validation to check validity of github pull request link match
-      const res = pullrequestLink.includes("issues")
-        ? await fetchGithubIssueInfo(orgName, repoName, issueId)
-        : await fetchGithubPRInfo(orgName, repoName, issueId);
+      const highlightType = highlightLink.includes("issues")
+        ? "issue"
+        : highlightLink.includes("pull")
+        ? "pull_request"
+        : "blog_post";
+      let res: any = {};
+      if (highlightType === "pull_request" || highlightType === "issue") {
+        const { apiPaths } = generateApiPrUrl(highlightLink);
+        const { repoName, orgName, issueId } = apiPaths;
+        setLoading(true);
+        // Api validation to check validity of github pull request link match
+        res = highlightLink.includes("issues")
+          ? await fetchGithubIssueInfo(orgName, repoName, issueId)
+          : await fetchGithubPRInfo(orgName, repoName, issueId);
+      } else {
+        res = await fetchDevToBlogInfo(highlightLink);
+      }
 
       if (res.isError) {
         setLoading(false);
 
-        toast({ description: "A valid Issue or Pull request Link is required", variant: "danger" });
+        toast({ description: "A valid Pull request, Issue or Blog Link is required", variant: "danger" });
```

Comment:
> We should provide context only DEV links work for now. Perhaps even leave a comment on this in case we get a request for hashnode or medium.

For Context

```suggestion
        toast({ description: "A valid Pull request, Issue or dev.to Blog Link is required", variant: "danger" });
```